src/libultra_code/libultra_internal.h

```diff
@@ -4,7 +4,7 @@
 
 // TODO: rename these
 //      SM64                OOT
-#define guMtxIdentF         func_80101B40
+#define guMtxIdentF         guMtxIdentF
```

Comment:
are these defines needed now that they are named properly? no clue what the purpose of them is. Also if the other one in here is a correct name why not rename that too I guess?

Reply:
That was there before I even started these files. I’ll fix all this when I get the chance today :)